ui-v2/tests/acceptance/dc/services/instances/show.feature

```diff
@@ -68,15 +68,12 @@ Feature: dc / services / instances / show: Show Service Instance
     And I see 3 of the serviceChecks object
     And I see 3 of the nodeChecks object
 
-    When I click tags on the tabs
-    And I see tagsIsSelected on the tabs
+    When I click tags&Meta on the tabs
+    And I see tags&MetaIsSelected on the tabs
```

Comment:
😬 ! Not sure if it would be good to sort the `&` out here, I suppose there nothing wrong here, just feels a bit wrong 😂  What do you think?

Reply:
Is there an easy way to fix this? If not, this works.